integration/kubernetes/multiMaster-embeddedJournal/master/alluxio-master-service.yaml.template

```diff
@@ -103,6 +105,4 @@ spec:
     role: alluxio-master
     app: alluxio
     release: alluxio
-    statefulset.kubernetes.io/pod-name: alluxio-master-2
----
-
```

Comment:
Also the output are all trimmed and lost the ending empty row. That's fine to me as we are removing these from the code repo.